src/main/java/org/cactoos/proc/BiProcOf.java

```diff
@@ -56,9 +56,22 @@ public BiProcOf(final Func<? super X, ?> func) {
      * @param func The bi function
      */
     public BiProcOf(final BiFunc<? super X, ? super Y, ?> func) {
+        this((BiProc<X, Y>) func::apply);
```

Comment:
@andreoss if we can avoid, it would be best not to use casting, it would be best to use `(x,y) -> { func.apply(x,y); }` for example

Reply:
@victornoel Done